<launch>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find vpdw_sim)/rviz/biped.rviz" required="true"/>
    <rosparam file="$(find vpdw_sim)/param/biped.yaml" command="load"/>
    <node pkg="vpdw_sim" type="vpdw_walker" name="vpdw_walker" output="screen"/>
    <node pkg="vpdw_sim" type="vpdw_plot" name="vpdw_plot" output="screen"/>
    <node pkg="vpdw_sim" type="ctc_controller" name="ctc_controller" output="screen"/>
</launch>
